lib/process-services-cloud/src/lib/task/task-filters/components/task-assignment-filter/task-assignment-filter.component.html

```diff
@@ -0,0 +1,20 @@
+<div fxLayout="row">
+    <mat-form-field>
+        <mat-select class="adf-task-assignment-filter"
+                    placeholder="{{ 'ADF_CLOUD_TASK_ASSIGNEMNT_FILTER.ASSIGNMENT_TYPE' | translate }}"
+                    [(ngModel)]="assignmentType"
+                    (ngModelChange)="onAssignmentTypeChange(assignmentType)">
+            <mat-option [value]="assignmentTypeList.currentUser">{{ 'ADF_CLOUD_TASK_ASSIGNEMNT_FILTER.ASSIGNED_TO_ME' | translate }} </mat-option>
+            <mat-option [value]="assignmentTypeList.unassigned">{{ 'ADF_CLOUD_TASK_ASSIGNEMNT_FILTER.UNASSIGNED' | translate }} </mat-option>
+            <mat-option [value]="assignmentTypeList.candidateGroups">{{ 'ADF_CLOUD_TASK_ASSIGNEMNT_FILTER.LIST_OF_CANDIDATE_USERS' | translate }} </mat-option>
```

Comment:
Is it `candidateGroups` or `LIST_OF_CANDIDATE_USERS` in the end? This is a little bit confusing